contrib/syslog-debun

```diff
@@ -31,25 +47,34 @@ pseao="ps -eao"
 dfk="df -k"
 dfh="df -h"
 duks="du -ks"
+grepq="grep -q"
+lddcmd="ldd"
 mount=mount
 varlimit=1000
 myplimit () { echo "Plimit query is not supported on this platform" >&3 ; }
 distpkgoffile () { echo "Package file search is not supported (yet) on this platform" >&3 ; }
 distpkgstatus () { echo "Package status query is not (yet) supported on this platform" >&3 ; }
 selftar="tar cf - ."
+gzipcmd="gzip -9"
 showdep="dpkg -S"
 pcapifparm=-i
 w=w
 vmstat=vmstat
 dmesg=dmesg
-tcpdump="tcpdump -p -s 1500 -w"
-getsyslogpids="pidof syslog-ng"
+tcpdumpcmd="tcpdump"
+tcpdumpopts="-p -s 1500 -w"
+getsyslogpids () { pidof syslog-ng ; }
 trace="strace -f"
-service_stop="/etc/init.d/syslog-ng stop"
-service_start="/etc/init.d/syslog-ng start"
-service_status="/etc/init.d/syslog-ng status"
-checkpid () { [ -d /proc/"$1" ]; }
-mywait () { jobs -p >${tmpdir}/sdn.jobs ; for i in `grep -v "\<$tailpid\>" ${tmpdir}/sdn.jobs` ; do wait $i ; done ; }
+initfile="/etc/init.d/syslog-ng"
+service_stop="${initfile} stop"
+service_start="${initfile} start"
+service_status="${initfile} status"
+#checkpid () { [ -d /proc/"$1" ]; }
```

Comment:
I just left it there for later reference. If the new code works well, the commented out parts will disappear in a later version. That was the logic behind it.
Usually support or customers simply download this script from somewhere other than git, so it seemed a good idea to leave some old and short code-snippets in the script, but commented out.